plone/app/discussion/browser/comments.pt

```diff
@@ -161,9 +161,7 @@
 
               </div>
 
-              <div class="comment-actions actions-delete"
-                   tal:condition="python:canDelete"
-              >
+              <div class="comment-actions actions-delete">
```

Comment:
There should still be a condition here because we don't want to render an empty container if neither button is available. Something like tal:condition="python:canDelete or (isDeleteOwnCommentAllowed and view.could_delete_own(reply))"